packages/merkle_tree/src/writeTree.ts

```diff
@@ -195,7 +199,10 @@ async function writeTree(blockHeight: number) {
       }
     }
   }
-  console.timeEnd("Get multisig guardians and pubkeys");
+  // Add the dev account
+  allAccounts.push(DEV_ACCOUNT);
```

Comment:
@DanTehrani reminder to add a few more accounts so we can play with this more easily in a deployed DB.

Reply:
noted!